float32[2] torque
